Packs/IntegrationsAndIncidentsHealthCheck/Scripts/GetFailedTasks/GetFailedTasks.py

```diff
@@ -1,10 +1,41 @@
 import demistomock as demisto  # noqa: F401
 from CommonServerPython import *  # noqa: F401
 
+BRAND = "Demisto REST API"
+
+
+def get_rest_api_instance_to_use(rest_api_instance_from_arg):
+    """
+        This function checks if there are more than one instance of demisto rest api.
+        Args:
+            rest_api_instance_from_arg (str) : rest api instance name
+
+        Returns:
+            Demisto Rest Api instance to use
+    """
+    allInstances = demisto.getModules()
+    number_of_rest_api_instances = 0
+    rest_api_instance_to_use = None
+    if rest_api_instance_from_arg is not None:
+        return rest_api_instance_from_arg
+    for instance_name in allInstances:
+        if allInstances[instance_name]['brand'] == BRAND:
+            rest_api_instance_to_use = instance_name
+            number_of_rest_api_instances += 1
+    if number_of_rest_api_instances > 1:
+        return_error("GetFailedTasks: This script can only run with a single instance of the Demisto REST API. "
+                     "Specify the instance name in the 'rest_api_instance' argument.")
+    elif number_of_rest_api_instances == 1:
+        return rest_api_instance_to_use
+    return rest_api_instance_from_arg
```

Comment:
@abaumgarten i meant that even if you have more than 1 in number_of_rest_api_instances, you should choose the given one if provided

Reply:
If I got that argument this would be returned (lines 19-20)

Reply:
so i guess it could be done in the main function:
rest_api_instance_to_use = get_rest_api_instance_to_use(rest_api_instance) if not rest_api_instance else rest_api_instance